### DISCUSSION TITLE
Emojis are not showed

### DISCUSSION THREAD

**Discussion by MathieuSoysal** [community/community#18824] [Other Feature Feedback, Questions, & Ideas] [answered]
With mobile version we cannot see emoji in the list:

![Screenshot_20220618-023804_GitHub.jpg](https://user-images.githubusercontent.com/43273304/174415360-e5271d7a-b041-405e-8aa0-09529e6dfec3.jpg)

**Comment by doguedogue** [comment]
I think because it is still in Beta 😁

**Comment by alamin-karno** [comment, marked as answer]
You have to update the new version of app then it will work for you

**Comment by Jazzo25** [comment]
Si

**Comment by Jazzo25** [comment]
Información
